DeepDive.md

```diff
@@ -36,12 +36,12 @@ This canister takes pre-signed transactions e.g. for `eth_sendRawTransaction` an
 
 ### JSON is not validated	
 
-This canister does not validate the JSON passed to the ETH service.  Registered API key providers should be aware of this in case the back end service is vulnerable to a bad JSON request body.  Registered API providers should be aware that there are methods in the Ethereum RPC API specification which give access to the ETH node keys.  Public service providers tend to block these, but registered API providers should ensure that they are not giving access to private keys or other privileged operations.	
+This canister does not validate the JSON passed to the ETH service.  Registered API key providers should be aware of this in case the back end service is vulnerable to a bad JSON request body.  Registered API providers should be aware that there are methods in the EVM RPC API specification which give access to the ETH node keys.  Public service providers tend to block these, but registered API providers should ensure that they are not giving access to private keys or other privileged operations.	
 
 ### Requests sent to service providers are subject to the service provider's privacy policy	
 
 Users should be aware of the privacy policy of the service provider to which their requests are sent as some service providers have stronger privacy guarantees.	
 
 ### Idempotency issues because of one HTTP outcall per node in the subnet	
 
-HTTP outcalls result in one HTTP outcall per node in the subnet and the results are then combined after filtering through a Transform function.  Some Ethereum RPC calls may not be idempotent or may vary which will cause the call to be reported as a failure as there will be no consensus on the result.  Furthermore this behavior may differ by service provider.  Users should be aware of this and use appropriate caution and/or a different solution.
+HTTP outcalls result in one HTTP outcall per node in the subnet and the results are then combined after filtering through a Transform function.  Some EVM RPC calls may not be idempotent or may vary which will cause the call to be reported as a failure as there will be no consensus on the result.  Furthermore this behavior may differ by service provider.  Users should be aware of this and use appropriate caution and/or a different solution.
```

Comment:
Why is "Transform" capitalized?

And what do you mean by "or may vary"? What's the difference to "not idempotent"?

Reply:
See the comment above; this is all from the original readme of the Phase 1 canister, which I moved into a different file because it was bugging me due to having so many grammatical and technical errors. We may need to remove or otherwise rewrite this entire document.